radixdlt/src/main/java/org/radix/database/exceptions/DatabaseException.java

```diff
@@ -15,18 +15,12 @@
  * language governing permissions and limitations under the License.
  */
 
-package org.radix.database.exceptions;
-
-import java.io.IOException;
-
-@SuppressWarnings("serial")
-public class DatabaseException extends IOException
-{
-	public DatabaseException () { super (); }
-
-	public DatabaseException (String arg0, Throwable arg1) { super (arg0, arg1); }
-
-	public DatabaseException (String arg0) { super (arg0); }
-
-	public DatabaseException (Throwable arg0) { super (arg0); }
-}
+package org.radix.database.exceptions;
+
+import java.io.IOException;
+
+public class DatabaseException extends IOException {
+	public DatabaseException(Throwable cause) {
```

Comment:
Might want to add an explanatory message here as well

Reply:
Sigh.  Yes.  Also added context message as well.